env/WCOSS2.env

```diff
@@ -86,40 +91,53 @@ elif [ $step = "gldas" ]; then
 
 elif [ $step = "eobs" ]; then
 
+    export OMP_PLACES=cores
+    export OMP_NUM_THREADS=$nth_eobs
```

Comment:
Why is this being set here?  This should be set in the ex-script immediately prior to the execution.
Also, what if `nth_eobs` is not the correct one?  `NTHREADS_GSI` is the correct one that is being supplied.

Reply:
I have corrected the NTHREADS settings (other comment) and removed the unneeded OMP_NUM_THREADS line.

Was there a question about OMP_PLACES too? That came from discussions with @RussTreadon-NOAA , @MichaelLueken-NOAA , and @GeorgeVandenberghe-NOAA .